.github/workflows/test-evm.yaml

```
@@ -119,10 +119,10 @@ jobs:
         run: node evm/deploy-its.js -m create2 -u -y
 
       - name: InterchainTokenService set hub
-        run: node evm/its.js --action setTrustedAddress --trustedChain axelar --trustedAddress axelar1xyz -y
+        run: node evm/its.js setTrustedAddress --trustedChain axelar --trustedAddress axelar1xyz -y
```

```suggestion
        run: node evm/its.js set-trusted-address axelar axelar1xyz -y
```
use kebab-case and convert args that are required to positional. See sui scripts for reference